python/agents/bidi-demo/app/main.py

```
@@ -0,0 +1,225 @@
+from google.adk.agents.live_request_queue import LiveRequestQueue
+from google.adk.agents.run_config import RunConfig, StreamingMode
+from google.adk.runners import Runner
+from google.adk.sessions import InMemorySessionService
+from google.genai import types
+
+# Load environment variables from .env file BEFORE importing agent
+load_dotenv(Path(__file__).parent / ".env")
+
+# Import agent after loading environment variables
+# pylint: disable=wrong-import-position
+from google_search_agent.agent import agent  # noqa: E402
```

Ruff (0.14.7)

Unused `noqa` directive (non-enabled: `E402`)

Remove unused `noqa` directive

(RUF100)